gordo_components/client/client.py

```diff
@@ -246,34 +249,29 @@ def download_model(self) -> typing.Dict[str, BaseEstimator]:
         """
         models = dict()
         for endpoint in self.endpoints:
-            resp = requests.get(f"{endpoint.endpoint}/download-model")
+            resp = self.session.get(f"{endpoint.endpoint}/download-model")
             if resp.ok:
                 models[endpoint.target_name] = serializer.loads(resp.content)
             else:
-                raise IOError(f"Failed to download model: '{resp.content}'")
+                raise IOError(f"Failed to download model: '{resp.content!r}'")
```

Comment:
Perhaps consider using `repr(resp.content)` instead of `!r`, as it is considered redundant according to `PEP 498`

Reply:
hmm, I was literally following mypy's suggestion

Reply:
I am not entirely sure, as I can not find any "best practices" on this exact case, but according to `PEP 498`: 
"The !s, !r, and !a conversions are not strictly required. Because arbitrary expressions are allowed inside the f-strings ...

... However, !s, !r, and !a are supported by this PEP in order to minimize the differences with str.format(). !s, !r, and !a are required in str.format() because it does not allow the execution of arbitrary expressions."

Reply:
Ya I don't know, but I can't tell if it doesn't fall into that because it's interpolating strings with bytes. 
`On Python 3 '{}'.format(b'abc') produces "b'abc'"; use !r if this is a desired behavior`

but using `repr()` is the same for me, and that works too :man_shrugging: